src/main/java/UITestFramework/CreateSession.java

```diff
@@ -44,7 +47,30 @@
 	Properties configProp = new Properties();
 	String OS;
 
+	public WebDriver getDriver() {
+        return driver;
+}
+
+@BeforeMethod
```

Comment:
@shivam198smart There is already @BeforeMethod present. Any specific reason to add this new one? If not, please remove it.

Reply:
Removed.